src/base/preferences.h

```diff
@@ -40,38 +41,50 @@ class QNetworkCookie;
 class QSize;
 class QTime;
 
-enum SchedulerDays
+namespace Scheduler
 {
-    EVERY_DAY,
-    WEEK_DAYS,
-    WEEK_ENDS,
-    MON,
-    TUE,
-    WED,
-    THU,
-    FRI,
-    SAT,
-    SUN
-};
+    Q_NAMESPACE
 
-namespace TrayIcon
-{
-    enum Style
+    enum class Days : int
     {
-        NORMAL = 0,
-        MONO_DARK,
-        MONO_LIGHT
+        EVERY_DAY,
```

Comment:
We use CamelCase names for `enum class` members, don't we?  In addition, they will look more friendly in the config file.

Reply:
Fixed.